autospec/specfiles.py

```diff
@@ -1495,8 +1495,8 @@ def write_meson_pattern(self):
 
         if config.config_opts['use_avx2']:
             self._write_strip('CFLAGS="$CFLAGS -m64 -march=haswell" CXXFLAGS="$CXXFLAGS -m64 -march=haswell " LDFLAGS="$LDFLAGS -m64 -march=haswell" '
-                              'meson --prefix /usr --libdir=/usr/lib64/haswell --buildtype=plain'
-                              '{0} {1} builddiravx2'.format(config.extra_configure, config.extra_configure64))
+                              'meson --prefix /usr --libdir=/usr/lib64/haswell --buildtype=plain ${0) '
```

Comment:
On second glance, I'm not understanding the addition to this line. Should it be ` {0} ` rather than ` ${0) `?

Reply:
Ah indeed, good catch. Looks like shell syntax snuck into the fingers.

Reply:
updated this PR with the catch fixed.